src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1378,17 +1349,20 @@ bool NetworkOPsImp::checkLastClosedLedger (
         // Temporary logging to make sure tiebreaking isn't broken
```

Comment:
I don't think this temporary logging is still useful, especially with the debug log above